winton_kafka_streams/kafka_streams.py

```diff
@@ -81,8 +81,8 @@ def is_created_or_running(self):
         def __str__(self):
             return self.name
 
-    def __init__(self, topology, kafka_config):
-        self.topology = topology
+    def __init__(self, topology_builder, kafka_config):
+        self.topology_builder = topology_builder
```

Comment:
If this is never read after `__init__` why store it in self?

Reply:
I can see no good reason. Maybe it was used previously. I'll remove this.